src/routes/components/button/+page.svelte

```diff
@@ -186,6 +198,14 @@ ${leftIcon || rightIcon ? `import { Close } from 'carbon-icons-svelte'` : ''}
 	</TabBar>
 {/snippet}
 
+{#snippet choose()}
+	<Typography></Typography>
+{/snippet}
+
+{#snippet sizes()}
+	<Typography></Typography>
+{/snippet}
+
```

Comment:
Yes, because the content is not done yet, but the template omits the sections if the snippets are missing. And it the sections are missing the helper text links won't work.